Alpha_wrap_3/doc/Alpha_wrap_3/alpha_wrap_3.txt

```diff
@@ -0,0 +1,370 @@
+into a Delaunay triangulation, and all finite triangles are tagged inside (grey).
+The current gate (green edge) popped out from the queue is alpha-traversable. The triangle adjacent
+to the gate is tagged outside when it does not intersect the input, and new alpha-traversable gates
+are pushed to the queue. When the adjacent triangle intersects the input, a new Steiner point (large green disc)
+is computed and inserted into the triangulation, all neighboring triangles are tagged inside,
+new alpha-traversable gates are pushed to the queue, and traversal is resumed.
+Grey edges depict the Delaunay triangulation. Blue edges depict the Voronoi diagram.
+Pink circles depict the empty circle of radius alpha. The output edges (dark blue) separate inside from outside triangles.
+\cgalFigureCaptionEnd
+
+\subsection aw3_guarantees Guarantees
+
+The algorithm is proven to terminate and to produce a 2-manifold triangulated surface mesh
+that strictly encloses the input data.
+The key element to the proof is that we wrap from outside to inside and never allow a cell
+that intersects the input to be flagged inside.
+Furthermore, both criteria that lead to refinement of the triangulation insert Steiner points
+that are guaranteed to break the cells in need of refinement and reduce the neighbor facets circumradii.
+
+Because the main refinement criterion is the insertion of an intersection between a dual Voronoi edge
+and an offset of the input, or the projection of a Voronoi vertex onto the offset of the input,
+the algorithm has similarities to popular meshing algorithms based on Delaunay filtering
+and refinement (see \ref Chapter_3D_Mesh_Generation).
+
+\section aw3_interface Interface
+
+Our algorithm takes as input a set of triangles in 3D, provided either as a triangle soup or
+as a triangle surface mesh, and two user-defined scalar parameters: the <em>alpha</em> and the <em>offset</em> values.
+It proceeds by shrink-wrapping and refining a 3D Delaunay triangulation starting from a loose bounding box of the input.
+The parameter <em>alpha</em> refers to the size of cavities or holes that cannot be traversed during wrapping,
+and hence to the final level of detail, as alpha acts like a sizing field in a common Delaunay
+refinement algorithm (\ref Chapter_3D_Mesh_Generation).
+The parameter <em>offset</em> refers to the distance between the vertices of the refined triangulation
+and the input, so that a large offset translates into a loose enclosing of the input.
+This second parameter offers a means to control the trade-off between tightness and complexity.
+
+The main entry point of the component is a global function that generates the alpha wrap;
+this function takes as input a polygon soup or a polygon mesh: `CGAL::alpha_wrap_3()`.
+There is no prerequisite on the input connectivity so that it can take an arbitrary triangle soup,
+with islands, self-intersections, or overlaps, as well as combinatorial or geometrical degeneracies.
```

Comment:
is islands clearly defined here? Mentioning closeness and connected components might not be either as you mention a soup.

Reply:
I think hand waving like this is clear enough given that we are not actually going to give an algorithm that aims to perform on specific configurations that would require more precise definitions.